lib/themes/base/tokens.ts

```diff
@@ -83,10 +83,14 @@ const makeColourGamut = () =>
 
 export const tokens: Tokens = {
 	breakpoints: {
-		mobile: 650,
-		tablet: 651,
-		desktop: 1001,
-		largeDesktop: 1201,
+		mobile: 0, // TODO: Check. Can this just be 0?
+		tablet: 768,
+		desktop: 1000,
+		largeDesktop: 1200,
+	},
+	contentWidth: {
+		large: 1300,
+		medium: 940,
```

Comment:
Should we be able to cap handheld devices between 800 to 1000 to a max with of 800px as per the existing rule?